app/api/items/add_item.go

```diff
@@ -112,6 +156,50 @@ func (in *NewItemRequest) canCreateItemsRelationsWithoutCycles(store *database.D
 	return count == 0
 }
 
+// swagger:operation POST /items items itemCreate
+// ---
+// summary: Create an item
+// description: >
+//
+//   Creates an item with parameters from the input data with `items.idDefaultLanguage` = `language_id`.
+//   Also it
+//
+//     * inserts a row into `users_items` with given `language_id`, `title`, `image_url`, `subtitle`, `description`,
+//
+//     * creates a new `groups_items` row with: `idItem` = `items.ID`, `idGroup` = `idGroupSelf` of the current user,
+//       `idUserCreated` = `users.ID` of the current user, `sFullAccessDate` = now(), `sCachedFullAccessDate` = now(),
+//       `bCachedFullAccess` = 1, `bOwnerAccess` = 1, `bManagerAccess` = 1.
+//
+//     * adds new relations for the parent and (optionally) children items into `items_items` and propagates `groups_items`.
+//
+//   The user should be an owner/manager of
+//
+//     * the `parent_item_id`,
+//     * `children` items (if any),
+//     * `unlocked_item_ids` items (if any),
+//
+//   and be an owner of `team_in_group_id` (if given),
+//   otherwise the "bad request" response is returned.
```

Comment:
"forbidden" would be more appropriate for most of these. Shouldn't we change?

Reply:
I don't think "forbidden" would be more appropriate here because it is just one of many possible errors of the form. If the validation fails, this error gets listed in `errors` among other form-related errors.